### PR TITLE
Added Travis config and Code Climate gem

### PR DESCRIPTION
**Pull request by dotemacs**
Travis is configured, but Code Climate isn't as it lies and says that
the repo on GitHub is private!

Upgraded a few other gems, as this project was created with rails 4.1.8 initially.

### CODE REVIEW

File: Gemfile [hunk 1]
```diff
@@ -1,15 +1,14 @@
...
 gem 'rails', '4.2.5.1'
 gem 'govuk_elements_rails', git: 'https://github.com/ministryofjustice/govuk_elements_rails.git', submodules: true
 # Use SCSS for stylesheets
-gem 'sass-rails', '~> 4.0.3'
+gem 'sass-rails', '4.0.5'
 # Use Uglifier as compressor for JavaScript assets
 gem 'uglifier', '>= 1.3.0'
 # Use CoffeeScript for .js.coffee assets and views
-gem 'coffee-rails', '~> 4.0.0'
+gem 'coffee-rails', '4.1.1'
```
Comment (novotnyjakub) [inline Gemfile]: I thought we'll be using Javascript?

File: Gemfile [hunk 2]
```diff
@@ -34,4 +33,9 @@ gem 'unicorn'
 group :development, :test do
   gem 'rspec-rails', '~> 3.0'
   gem 'rubocop', require: false
+  gem 'rubocop-rspec', require: false
+end
+
+group :test do
+  gem 'codeclimate-test-reporter', require: nil
```
Comment (novotnyjakub) [inline Gemfile]: Do we need this if Github does this as hooks? Or is it something extra? I've never used this gem, so I'm curious.

File: Gemfile (hunk 1 above)
Comment (dotemacs) [inline Gemfile]: Good point. I was just driven by the fact that it's there in the Gemfile and that it's outdated.

I'll remove it now.

File: Gemfile (hunk 2 above)
Comment (dotemacs) [inline Gemfile]: It's CodeClimate setup for usage on Travis CI.

See: https://docs.travis-ci.com/user/code-climate/

**Comment by colinbruce** [comment]
Travis config can be fun!  Once code-climate accepts the repo, you'll need a key in the travis YML to get it working!

Looks fine otherwise! Merge at will!

**Pull request merged**